packages/web-components/README.md

```diff
@@ -0,0 +1,33 @@
+# Web Components
+
```

Comment:
For the component call-outs, for example `fast-element` when used at the beginning of each paragraph,  should we make them links to the actual package documentation?

Reply:
We don't have documentation landing pages that are meaningful for each package, unless we want to link the API doc landing page. I could change it to that, but probably not what someone is looking for if they want to get started.